sdk/cognitiveservices/arm-cognitiveservices/README.md

```diff
@@ -19,8 +19,9 @@ npm install @azure/arm-cognitiveservices
 
 ##### Install @azure/ms-rest-nodeauth
 
+- Please install minimum version of `"@azure/ms-rest-nodeauth": "^3.0.0"`.
```

Comment:
I would be a little more explicit here. Something like

> To authenticate, you will need the credentials created from using the [@azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) library. Please ensure that you are using version 3.0.0 or above. The older versions of the `@azure/ms-rest-nodeauth` are not compatible with the latest version of this library.

Reply:
They are auto generated by autorest.typescript. If we need to improve the description, autorest plugin needs to be updated.

Reply:
Yep, this is generated by `autorest.typescript`, Amar added this message in his most recent update to the 4.x series.